src/analysis.ts

```diff
@@ -482,3 +446,81 @@ export async function analyzeGit(options: GitOptions, requestOptions?: RequestOp
 
   return result;
 }
+
+interface CreateBundleFromFolders extends FolderOptions {
```

Comment:
nit/optional: not an ideal or descriptive enough name IMO, should be something like `CreateBundleFromFoldersOptions`.